src/config.ts

```diff
@@ -2935,4 +2941,53 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://axelarscan.io/tx/{txHash}',
 	},
+  {
+		rpc: 'https://rpc-fetchhub.fetch.ai:443',
+		rest: 'https://rest-fetchhub.fetch.ai',
+		chainId: 'fetchhub-4',
+		chainName: 'Fetch Hub',
+		stakeCurrency: {
+			coinDenom: 'FET',
+			coinMinimalDenom: 'afet',
+			coinDecimals: 18,
+			coinGeckoId: 'fetch-ai',
+			coinImageUrl: window.location.origin + '/public/assets/tokens/fet.png',
+		},
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('fetch'),
+		currencies: [
+			{
+				coinDenom: 'FET',
+				coinMinimalDenom: 'afet',
+				coinDecimals: 18,
+				coinGeckoId: 'pool:nanomobix',
```

Comment:
this alternative price via pool hasn't been defined. Please create a pool, add liquidity, and define pool:nanomobix using that new pool
https://docs.osmosis.zone/integrate/frontend.html#how-to-specify-asset-price-oracle-on-osmosis-zone-liquidity-pool

Reply:
It looks like the MOBX CoinGeckoId is declared under FET, and the FET CoinGeckoId declared under MOBX, so these need to be switched

Reply:
Won't include MOBX this time around.